examples/va/hci/edpm-pre-ceph/values.yaml

```diff
@@ -27,6 +27,9 @@ data:
         edpm_iscsid_image: '{{ registry_url }}/openstack-iscsid:{{ image_tag }}'
         edpm_logrotate_crond_image: '{{ registry_url }}/openstack-cron:{{ image_tag }}'
         edpm_network_config_hide_sensitive_logs: false
+        edpm_network_config_os_net_config_mappings:
+          edpm-copmpute:
```

Comment:
typo

Reply:
Isn't [that](https://github.com/openstack-k8s-operators/edpm-ansible/blob/23473519a2b52515a4d4bdb04d264828774087c9/roles/edpm_network_config/tasks/network_config.yml#L105-L106) the name of the var?  And [this](https://github.com/openstack-k8s-operators/edpm-ansible/blob/23473519a2b52515a4d4bdb04d264828774087c9/plugins/modules/edpm_os_net_config_mappings.py#L119-L133) is how it's handled.